dashboards/victoriametrics-cluster.json

```diff
@@ -3777,164 +3782,147 @@
               },
               "editorMode": "code",
               "exemplar": true,
-              "expr": "max(\n    rate(vm_cache_misses_total{job=~\"$job\", instance=~\"$instance\"}[$__rate_interval])\n    /\n    rate(vm_cache_requests_total{job=~\"$job\", instance=~\"$instance\"}[$__rate_interval])\n) by(type)",
+              "expr": "sum(vm_rpc_vmstorage_is_read_only{job=~\"$job_insert\", instance=~\"$instance\"}) by(instance, addr) > 0",
               "format": "time_series",
-              "hide": false,
               "interval": "",
               "intervalFactor": 1,
-              "legendFormat": "__auto",
+              "legendFormat": "{{instance}} => {{addr}}",
               "range": true,
               "refId": "A"
             }
           ],
-          "title": "Cache miss ratio ($instance)",
+          "title": "Storage in readonly status for vminsert ($instance)",
           "type": "timeseries"
         },
         {
           "datasource": {
             "type": "prometheus",
             "uid": "$ds"
           },
+          "description": "Shows rate of samples [deduplication](https://docs.victoriametrics.com/#deduplication)  according to the configured `-dedup.minScrapeInterval`.\n\nShould be empty if `-dedup.minScrapeInterval` isn't set.\n\nDeduplication happens in two places:\n* `type=\"merge\"` - during [background merges](https://docs.victoriametrics.com/#storage) by vmstorages. Deduplication during merges is permanent.\n* `type=\"select\"` - during [read queries](https://docs.victoriametrics.com/keyconcepts/#query-data) by vmselects. Deduplication happens in-flight and has no permanent effect.\n\nThe more samples need to be deduplicated, the higher will be resource usage on vmselects or vmstorages.",
```

Comment:
Should this also mention `-downsampling.period`?
Downsampling also processes data during background merge, so it will affect metric with `type="merge"` as well.

Reply:
tx! see updated